### PR TITLE
Text and style updates

### PR DESCRIPTION
**Pull request by abhinayagarwal**
This PR updates the text to make it generic. Phrases like `Getting Started with JavaFX 13` has been updated to only say `Getting Started with JavaFX`, which is more generic and sounds better.

Also, updates styling for few HTML elements.

### CODE REVIEW

File: gradle.html [hunk 1]
```diff
@@ -69,5 +69,5 @@ <h2>Run HelloWorld using Gradle</h2>
 
 <div class="alert alert-warning">
     <strong>Note: </strong>
-    Make sure to use gradle version 4.10+ and set the JAVA_HOME environment variable to JDK <span class="JDK_MAJOR">11</span>.
+    Make sure to use gradle version 4.10+ and set the JAVA_HOME environment variable to correct JDK location.
```
Comment (jperedadnr) [inline gradle.html]: Probably we need a higher gradle version now... at least 6?

File: ide-eclipse.html [hunk 2]
```diff
@@ -1,25 +1,25 @@
-<h2>JavaFX <span class="JFX_MAJOR">11</span> and Eclipse</h2>
+<h2>JavaFX and Eclipse</h2>
 
-<p>This section explains how to use Java <span class="JDK_MAJOR">11</span> and JavaFX <span class="JFX_MAJOR">11</span> from Eclipse.
-    Version: 2019-03 (4.11.0) was used for the following screenshots.
+<p>
+    This section explains how to create a JavaFX application in Eclipse.
+    JavaFX 12.0.2 and Eclipse 2019-03 (4.11.0) was used for the IDE screenshots.
```
Comment (jperedadnr) [inline ide-eclipse.html]: At some point we need to update the IDEs and the screenshots as well

File: ide-eclipse.html [hunk 3]
```diff
@@ -1,25 +1,25 @@
...
 
 <p>
     To get Java 12 support, install the patch from the Eclipse MarketPlace: <a href="https://marketplace.eclipse.org/content/java-12-support-eclipse-2019-03-411" target="_blank">Java 12 Support for Eclipse 2019-03 (4.11)</a>.
 </p>
 
 <p>
-    Download an appropriate <a href="https://jdk.java.net/12/">JDK <span class="JDK_MAJOR">11</span></a> for your operating system.
-    Make sure <kbd>JAVA_HOME</kbd> is properly set to the Java <span class="JDK_MAJOR">11</span> installation directory. 
+    Download an appropriate JDK for your operating system.
+    Refer to <a href="#install-java">Install Java</a> section for more information.
```
Comment (jperedadnr) [inline ide-eclipse.html]: Big change... imo JAVA_HOME, PATH, JDK 11+, are required terms, even if we repeat them elsewhere...

File: ide-eclipse.html [hunk 4]
```diff
@@ -309,7 +302,7 @@ <h5>1. Create a Gradle project</h5>
 
     <a href="images/ide/eclipse/gradle/eclipse00.png" target="_blank"><img src="images/ide/eclipse/gradle/eclipse00.png" alt="Create a Gradle project"/></a>
 
-    Provide a name to the project, like <kbd>HelloFX</kbd> and a location for the project. Then point Java Home to JDK <span class="JDK_MAJOR">11</span>.
+    Provide a name to the project, like <kbd>HelloFX</kbd> and a location for the project. Then point JAVA_HOME to JDK <span class="JDK_MAJOR">11</span>.
```
Comment (jperedadnr) [inline ide-eclipse.html]: <kbd>JAVA_HOME</kbd>

File: ide-netbeans.html [hunk 5]
```diff
@@ -1,41 +1,30 @@
-<h2>JavaFX <span class="JFX_MAJOR">11</span> and NetBeans</h2>
+<h2>JavaFX and NetBeans</h2>
 
-<p>This section explains how to use Java <span class="JDK_MAJOR">11</span> and JavaFX <span class="JFX_MAJOR">11</span> from NetBeans. 
-    Apache NetBeans 11.1 was used for the following screenshots. It can be downloaded from
-    <a href="https://netbeans.apache.org/download/nb111/nb111.html" target="_blank">here</a>.
+<p>
+    This section explains how to create a JavaFX application in NetBeans.
+    JavaFX 12.0.2 and Apache NetBeans 11.1 was used for the IDE screenshots.
```
Comment (jperedadnr) [inline ide-netbeans.html]: s/was/were

File: ide-eclipse.html (hunk 2 above)
Comment (jperedadnr) [reply ide-eclipse.html]: s/was/were

File: install-java.html [hunk 6]
```diff
@@ -20,12 +20,12 @@ <h2>Install Java <span class="JDK_MAJOR">11</span></h2>
...
-    You need to set the <b>JAVA_HOME</b> environment variable to a JDK <span class="JDK_MAJOR">11</span> installation directory. You can follow 
+    You need to set the <b>JAVA_HOME</b> environment variable to the JDK installation directory. You can follow 
     <a href="http://www.baeldung.com/java-home-on-windows-7-8-10-mac-os-x-linux" target="_blank">this guide</a> to set JAVA_HOME for your platform.
 </p>
 
 <div class="alert alert-warning">
     <strong>Note: </strong>
-    If you have newer or older versions of Java installed along with JDK <span class="JDK_MAJOR">11</span>, you need to make sure that the JAVA_HOME
-    environment variable points to JDK <span class="JDK_MAJOR">11</span>.
+    If your system has multiple versions of JDK installed, you need to make sure that the JAVA_HOME
```
Comment (jperedadnr) [inline install-java.html]: <kdb>JAVA_HOME</kdb>

File: gradle.html (hunk 1 above)
Comment (abhinayagarwal) [reply gradle.html]: We need a Gradle -> Java support table :)

File: ide-eclipse.html (hunk 3 above)
Comment (abhinayagarwal) [reply ide-eclipse.html]: How about "Download an appropriate JDK for your operating system and set `JAVA_HOME` to this JDK directory" ?

File: gradle.html (hunk 1 above)
Comment (jperedadnr) [reply gradle.html]: Anyway, it doesn't make sense keeping a very old gradle version. JDK 13 requires gradle 5? and 14 6.3.

File: ide-eclipse.html (hunk 3 above)
Comment (jperedadnr) [reply ide-eclipse.html]: ok

File: gradle.html (hunk 1 above)
Comment (kevinrushforth) [reply gradle.html]: JDK 13 requires gradle 6.0 or later. JDK 14 will require gradle 6.3.

Comment (abhinayagarwal) [reply gradle.html]: I have added a table to show our recommended minimum Gradle version against each JDK.

**Pull request merged**